cypress/e2e/hub/signature-keys.cy.ts

```diff
@@ -1,12 +1,42 @@
 import { SignatureKeys } from './constants';
+import { pulpAPI } from '../../support/formatApiPathForHub';
 
 describe('Signature Keys', () => {
   before(() => {
     cy.hubLogin();
   });
 
-  it('it should render the signature keys page', () => {
+  it('should render the signature keys page', () => {
     cy.navigateTo('hub', SignatureKeys.url);
     cy.verifyPageTitle(SignatureKeys.title);
   });
+  it('should show no content when no data', () => {
+    cy.contains('No signature keys yet').should('be.visible');
+  });
+  it('should contain correct table headers', () => {
+    // galaxykit not supporting signature keys yet, mocking instead
```

Comment:
it's not galaxykit, it's that the api instance is running 4.7.1 without signing services configured

Reply:
Fixed the comment